_query-dsl/joining/parent-id.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Parent ID
+parent: Joining queries
+nav_order: 40
+---
+
+# Parent ID query
+
+The `parent_id` query returns child documents whose parent document has the specified ID. You can establish parent-child relationships between documents in the same index by using a [join]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/join/) field type.
```

Comment:
```suggestion
The `parent_id` query returns child documents whose parent documents have the specified ID. You can establish parent-child relationships between documents in the same index by using a [join]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/join/) field type.
```

Reply:
The parent:child is a 1:many relationship so leaving singular.